<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cloneSensor</class>
 <widget class="QDialog" name="cloneSensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clone sensor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,3,0">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>New sensor name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sensorName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="clearSensorData">
     <property name="text">
      <string>Clear data (not working)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="sensorShort">
       <property name="text">
        <string>short</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="sensorFrame">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>begin date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="sensorBegin">
            <property name="displayFormat">
             <string>yyyy-MM-dd hh:mm:ss</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>end date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="sensorEnd">
            <property name="displayFormat">
             <string>yyyy-MM-dd hh:mm:ss</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,1">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Timezone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sensorTz">
            <property name="minimum">
             <number>-12</number>
            </property>
            <property name="maximum">
             <number>12</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>cloneSensor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>cloneSensor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
